great_expectations/expectations/metrics/map_metric_provider/column_map_condition_auxilliary_methods.py

```diff
@@ -84,11 +84,20 @@ def _pandas_column_map_condition_values(
     ]
 
     result_format = metric_value_kwargs["result_format"]
+    unexpected_metrics_with_values: bool = result_format.get(
+        "unexpected_metrics_with_values", True
+    )
 
     if result_format["result_format"] == "COMPLETE":
-        return list(domain_values)
+        if not unexpected_metrics_with_values:
+            return []
+        else:
+            return list(domain_values)
 
-    return list(domain_values[: result_format["partial_unexpected_count"]])
+    elif not unexpected_metrics_with_values:
+        return []
+    else:
+        return list(domain_values[: result_format["partial_unexpected_count"]])
```

Comment:
I'm not sure if this file is the place to implement this change, but would it be possible to suppress the `unexpected_list` in the output rather than an empty list? The concern is that an empty list would communicate that there are no unexpected values and could potentially be confusing.

So for example the result (when your new parameter is implemented) would be:
```
"unexpected_index_list": [
      {
            "__id" : [14997, 15600, ...],
      }
],
```
instead of:
```
"unexpected_index_list": [
      {
            "__id" : [14997, 15600, ...],
      }
],
"unexpected_list": [ ]
```

Reply:
I have made changes on `_format_map_output` function : In case of `exclude_unexpected_values=False`, the following attributes are suppressed : 

- `unexpected_list`
- `partial_unexpected_list`
- `partial_unexpected_counts` : _It contains values too..._